resources/waterheater.rb

```diff
@@ -9,7 +9,7 @@
 
 class Waterheater
   def self.apply_tank(model, runner, space, fuel_type, cap, vol, ef,
-                      re, t_set, oncycle_p, offcycle_p, ec_adj, nbeds, dhw_map, sys_id)
+                      re, t_set, oncycle_p, offcycle_p, ec_adj, nbeds, dhw_map, sys_id, has_desuperheater, related_clg_coil)
```

Comment:
If we just have a single argument, I suggest calling it desuperheater_clg_coil instead of related_clg_coil.